src/react-components/input/CheckboxInput.js

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef, memo } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import styles from "./CheckboxInput.scss";
+
+export const CheckboxInput = memo(
+  forwardRef(({ className, labelClassName, label, description, disabled, ...rest }, ref) => {
+    return (
+      <label className={classNames(styles.checkboxInput, { [styles.disabled]: disabled }, className)}>
+        <input type="checkbox" disabled={disabled} ref={ref} {...rest} />
+        <div className={classNames(styles.checkmark)} />
+        {label && (
```

Comment:
I wonder if this label ought to be its own component, such that this becomes a `LabeledCheckboxInput` that combines a `Label` and a `CheckboxInput`. 

Maybe not worth breaking this down unless we wanted to re-use the "label" portion.

Reply:
Good point for reuse on labels, I'll keep that in mind for future inputs (then redo RadioInput, CheckboxInput etc) -- but keeping this as is for now since it follows the RadioInput component standard.